LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/BasicsStation/DownlinkBasicsStationMessage.cs

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace LoRaTools.LoRaPhysical
+{
+    using System;
+    using LoRaWan;
+
+    /// <summary>
+    /// JSON of a Downlink message for the Basic Station.
```

Comment:
I don't see anything JSON-specific in this class. I believe the real conversion is being done in `DownstreamSender.Message`?

Reply:
I changed the comment,